src/parameters/all_parameters.cpp

```diff
@@ -267,6 +267,15 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       "Dissipative numerical flux. "
                       "Choices are <symm_internal_penalty | bassi_rebay_2 | central_visc_flux>.");
 
+    prm.declare_entry("non_physical_behavior", "return_big_number",
+                      dealii::Patterns::Selection("return_big_number | abort_run | print_warning"),
+                      "Behavior when a nonphysical result is detected in physics, "
+                      "For example negative density or NaN. "
+                      "return_big_number will return BIG_NUMBER without any warnings "
```

Comment:
```suggestion
                      "return_big_number will return BIG_NUMBER (i.e. will set density to BIG_NUMBER) without any warnings "
```

Reply:
I've changed the comment to "return_big_number will set the quantity to BIG_NUMBER without any warnings", since density is not the only value that is changed.